save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ScoreCard.kt

```diff
@@ -35,7 +40,7 @@ external interface ScoreCardProps : Props {
  * @return ReactElement
  */
 @Suppress("TOO_LONG_FUNCTION", "LongMethod")
-fun scoreCard() = FC<ScoreCardProps> { props ->
+private fun scoreCard() = FC<ScoreCardProps> { props ->
```

Comment:
```suggestion
val scoreCard = FC<ScoreCardProps> { props ->
```
I meant simply like this

Reply:
To my mind it might be convenient to keep private fun in order if we would like to add params later

Reply:
I think it's almost always more correct to pass all parameters as props, for older components we have parameters because of our lack of understanding of React back then :) I think we can safely assume that these new components can be extended using props if it'll be needed

Reply:
Some things like style can be left as arguments (for example as it is in `cardComponent`) because it defines type of a component, I do not think it should be in props. For example we might want scoreCard component to have a different background or different shape. In this case it might be good to pass a param like `hasBg` or `isBoarded`. To my mind it should not be in props cause props seem to be more functional set of attributes.

Reply:
Ok, that makes sense